### DISCUSSION TITLE
What is easiest way to get all metrics from a old server and backfill a new server?

### DISCUSSION THREAD

**Discussion by ForceConstant** [prometheus/prometheus#12491] [Q&A] [answered]
I have a new prometheus instance running, and would like to back fill metrics using all the metrics on an old instance. My various ideas that didn't work quite were to extract with python, and write to new server, but couldn't actually figure out to write with python. Then I see that you can backfill with openmetrics data, but couldn't figure out how to extract old data to openmetrics format. Any other ideas?

**Comment by roidelapluie** [comment, marked as answer]
You should just be able to copy over the blocks (the folders in the old
server except Wal and chunks_head) to the data directory of the new server.

**Comment by ForceConstant** [comment]
thanks that worked.
